File: all_star_15.py
def rotate(str_):
    arr = list(str_)
    count = 0
    new_arr = []
    while count < len(arr):
        first = arr.pop(0)
        arr.append(first)
        new_word = ''.join(arr)
        new_arr.append(new_word)
        count += 1
    return new_arr

File: allStar15.js
function rotate(str){
    let arr = str.split('');
     let count = 0;
     let newArr = [];
     while (count < arr.length){
       let first = arr.shift();
       arr.push(first);
       let newWord = arr.join('');
       newArr.push(newWord);
       count++;
     }
     return newArr;
   }